financial-connections/res/values/strings.xml

```diff
@@ -114,6 +114,10 @@
     <string name="stripe_networking_signup_cta_save">Save to Link</string>
     <string name="stripe_networking_signup_cta_negative">Not now</string>
     <string name="stripe_networking_signup_title">Save your account to Link</string>
+    <string name="stripe_networking_link_login_warmup_email_label">Continue as</string>
+    <string name="stripe_networking_link_login_warmup_skip">Not you? <annotation clickable="skip_login">Continue without signing in</annotation></string>
+    <string name="stripe_networking_link_login_warmup_description">It looks like you have a Link with Stripe account. Signing in will let you quickly access your saved bank accounts.</string>
```

Comment:
I'd double check the "with Stripe account" part:
- https://git.corp.stripe.com/stripe-internal/stripe-js-v3/blob/19a2caa3/src/linkedAccounts/inner/components/panes/NetworkingLinkLoginWarmup/messages.ts#L15

Reply:
Good catch https://github.com/stripe/stripe-android/pull/6185/commits/b65c2d7fac589accc9d3761ec3ef93efa39d136f